source/configure_project/frontmatter/index.html.md.erb

```diff
@@ -173,5 +183,14 @@ title: My beautiful page
 ---
 ```
 
+### description
+
+Description for the page, used by search engines and when shared on social media.
+
+```yaml
+---
+description: One of the best pages on the Internet.
```

Comment:
```suggestion
description: Configuring page settings in the GDS Technical Documentation Template.
```

Thought it might be best to use a more real example, to help users set an appropriate description. Hope that's ok!